### DISCUSSION TITLE
Validation of variables for an specific project and not for the entire repo

### DISCUSSION THREAD

**Discussion by JACONDE** [Dynatrace/dynatrace-configuration-as-code#207] [Archive] [answered]
It looks monaco is always validating that all projects variables exists, although you are trying to execute only one of many.
This is a real challenge when trying to create releases in Azuredevops (for instance) each one of the release must have ALL variables used in ALL projects, even do they are not needed for an specific project.
It will good to remove that dependency, and only validate the variables that are used for the specific projects or projects executed and with dependencies.

**Comment by tatomir146** [comment, marked as answer]
This relates to #71
